### DISCUSSION TITLE
Pen only working when pressing a button

### DISCUSSION THREAD

**Discussion by ArcovanBreda** [xournalpp/xournalpp#5190] [Q&A]
Hi all! I'm using this application for note taking and would think it is great! I have a dell laptop and a dell pen. I got the pen working, however not by touching my touchscreen but only when I touch the touchscreen AND touch one of the two buttons of my pen. This is ofcourse annoying and would give me cramps on longer note taking journeys. Do you have any idea how this can be resolved?

**Comment by rolandlo** [comment]
That sounds strange and annoying indeed. Can you please post a screenshot of your settings in the input system tab of menu `Edit > Preferences`?
